README.md

```diff
@@ -283,3 +283,43 @@ To note: the JenkinsCI server will almost always have a green check mark in the
 Check out the [contributing guidelines](.github/CONTRIBUTING.md)!
 
 [topdoc-link]: https://github.com/Topdoc/topdoc/wiki
+
+## Releasing
+
+We are currently releasing this pacakage on `npm`. We are also releasing an internal version of this package for Adobe on an internal instance of Artifactory. 
+
+First, prepare the package for release by bumping the version (minor version bump by default), committing the changes, tagging the repo, and pushing the changes and tags up to GitHub (origin remote). We have a command that does this for you. 
```

Comment:
Assuming my suggested change around parsing the remote is incorporated, let's make sure to remove the note around the origin remote in here.